gitlabform/processors/group/group_members_processor.py

```
@@ -289,8 +295,8 @@ def _process_users(
                 try:
                     group.members.delete(user_id)
                 except GitlabDeleteError as delete_error:
-                    warning(f"Member '{user}' could not be deleted: {delete_error}")
-                    pass
+                    error(f"Member '{user}' could not be deleted: {delete_error}")
+                    raise delete_error
```

just double checking if this will have any side effects with processing as we are changing the behaviour from passing to raising exception.

---

Looks good to me otherwise.

---

@SachinKSingh28 yes it will but I think rather than hiding the error it should be raised - that is consistent with the Project Members processor